### PR TITLE
Avoids receiving warning about galera state

### PR DESCRIPTION
**Pull request by ced1check**
Since 1.36, I started receiving alerts about this on all my servers (7 running Ubuntu and MariaDB 10.6.7 / 10.6.9).
I had to edit the configuration to remove the warning.

Galera status is a text like Primary or Donor/Desync, however the alert seems to expect a number?

### CODE REVIEW

**Comment by thiagoftsm** [comment]
Hello @ced1check ,

Thank you for your contribution! Please, sign our [cla](https://github.com/netdata/netdata/pull/13550#issuecomment-1222036044), because without your signature we cannot merge your PR.

Best regards!

File: health/health.d/mysql.conf [hunk 1]
```diff
@@ -160,7 +160,7 @@ component: MySQL
 component: MySQL
      calc: $wsrep_cluster_status
     every: 10s
-     crit: $mysql_galera_cluster_state != nan AND $this != 0
+     crit: $mysql_galera_cluster_state != nan AND $this != 0 AND $this != 'Primary'
```
Comment (ilyam8) [inline health/health.d/mysql.conf:163]: That is wrong, `$this` is an integer value. 

---

The current implementation is to decode the status as an integer, see the [function](https://github.com/netdata/go.d.plugin/blob/faa01a27a40818eeef5b849ae599353ab000dc48/modules/mysql/collect_global_status.go#L65-L84).

The alarm is critical when the wsrep cluster status is not 0 (not primary/quorum present).

**Review by ilyam8 — CHANGES_REQUESTED** [review]

File: health/health.d/mysql.conf (hunk 1 above)
Comment (ced1check) [reply health/health.d/mysql.conf:163]: Maybe it's wrong but if I don't apply that change for non integer value I keep receiving an alert for this while the graph shows a flat 0. I don't say it's the right way to fix it but having no time to dig into source code I fixed it that way on 7 servers. 

So let me rephrase all this: I get a critical alarm as seen on screenshot if I don't make that change.

Comment (ilyam8) [reply health/health.d/mysql.conf:163]: The only screenshot in the issue is the Cloud Alarms tab screenshot. Can you share a screenshot from the local dashboard?

<details>
<summary>e.g. screenshot click</summary>

Screenshot 2022-08-22 at 15 44 45

</details>

Comment (ced1check) [reply health/health.d/mysql.conf:163]: critical_alert

Comment (ilyam8) [reply health/health.d/mysql.conf:163]: Thanks, but I asked for a screenshot from the local dashboard, not Cloud.

Comment (ced1check) [reply health/health.d/mysql.conf:163]: I don't use the local dashboard because access to those servers is simply not available.
